tests/testthat/test-build-search-docs.R

```diff
@@ -21,6 +21,5 @@ test_that("build_search() builds the expected search.json", {
   build_news(pkg)
   build_home(pkg)
   build_sitemap(pkg)
-  build_search(pkg)
-  expect_snapshot_file(path(tmp, "search.json"))
+  expect_snapshot(build_search_index(pkg))
```

Comment:
Using `str()` makes the output a bit easier to read. This is also one of the few cases where I'd use `expect_snapshot_output()` since the code is not actually that helpful.